index.js

```diff
@@ -0,0 +1,8 @@
+
+'use strict';
+
+const PORT = process.env.PORT;
+require('dotenv').config();
```

Comment:
Lines 4 and 5 need to be swapped, you need to configure the .env before you use the PORT